open_discussions/views.py

```diff
@@ -62,6 +62,9 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
         user_email = user.email
     js_settings = {
         "gaTrackingID": settings.GA_TRACKING_ID,
+        "gaChannelTrackers": dict(
```

Comment:
I have two concerns about this:
1) this list could grow large
2) this exposes the names of our channels to people who aren't members.

I'd suggest that we should consider returning this in the channel api instead, since that will be limited to channels the user can see.